package.json

```diff
@@ -22,7 +22,8 @@
 	"dependencies": {
 		"bourbon": "4.2.3",
 		"sass-rem": "^1.2.4",
-		"swarm-constants": "0.1.16"
+		"swarm-constants": "0.1.16",
+    "swarm-animation": "https://github.com/meetup/swarm-animation.git"
```

Comment:
indentation nitpick. 

and I'm unsure whether this should be a `dependency` or `peerDependency`? I believe `peerDependency` you wont have the same dependency installed more that once. Lets confirm with @mattkime